subprojects/code-quality/src/main/groovy/org/gradle/api/plugins/quality/FindBugs.java

```diff
@@ -328,12 +329,26 @@ public FileTree getSource() {
     @SkipWhenEmpty
     @PathSensitive(PathSensitivity.RELATIVE)
     @InputFiles
+    public FileCollection getCandidateClasses() {
```

Comment:
The javadoc is hidden in the diff.

But on second thought about this, I think filtering by `.class` files might be wrong here. I think nothing stops someone from doing something like:

```
findbugsMain {
   classes = files("someJar.jar")
}
```
And this would work.  I took your advice and renamed the method and made it less visible.